scripts/push_docs_to_branch.sh

```diff
@@ -14,8 +14,8 @@ set -Eeuo pipefail
 
```

Comment:
we also need to port the changes from [this PR](https://github.com/RasaHQ/rasa/pull/9266/files) into `rasa-sdk` (and worth double checking that the `|| exit 1` are present everywhere)